File: back-end/GeekSpot.Domain/Entities/Comentario.cs
using System.ComponentModel.DataAnnotations;
using static GeekSpot.Utils.Biblioteca;

namespace GeekSpot.Domain.Entities
{
    public class Comentario
    {
        [Key]
        public int ComentarioId { get; set; }

        // Fk (De lá pra cá);
        public int ItemId { get; set; }
        public Item? Itens { get; set; }

        // Fk (De lá pra cá);
        public int UsuarioId { get; set; } // Usuário que fez a pergunta;
        public Usuario? Usuarios { get; set; }

        public string? Mensagem { get; set; } = null;
        public string? Resposta { get; set; } = null;

        public int IsAtivo { get; set; } = 1;
        public DateTime? DataEnvio { get; set; } = HorarioBrasilia();
    }
}

File: front-end/utils/types/comentario.ts
export default interface iComentario {
    comentario: {
        comentarioId: number;
        itemId: number;
        itens: {
            itemId: number;
            nome: string;
        };
        usuarioId: number;
        usuarios: {
            usuarioId: number;
            nomeUsuarioSistema: string;
        };
        mensagem: string;
        resposta: string | null;
        isAtivo: number;
        dataEnvio: Date;

        erro: boolean | null;
        codigoErro: number | null;
        mensagemErro: string | null;
    }[]
}